_includes/cloud-query/cloud-query-open-schema-browser.md

```diff
@@ -0,0 +1,6 @@
+* Click the button in the top left labeled **SCHEMA BROWSER**
+
+When the schema browser is open, you will see a list of tables in your database on the right side of the screen. 
```

Comment:
Same here -- no sticking explanations in the middle of procedures. It confuses the issue.